src/headlessTask/index.js

```diff
@@ -8,24 +8,31 @@ import SharedPreferences from 'react-native-shared-preferences';
 
 import 'moment/min/locales';
 
-function getNoticeIds(matchingContexts, eventMessageFromChromeURL) {
-  return matchingContexts
-    .map((res) => {
-      const addWWWForBuildingURL = `www.${eventMessageFromChromeURL}`;
-
-      if (res.xpath) {
-        return;
-      }
+let matchingContexts = [];
+let HTML = '';
+let noticeIds = [];
+
+async function getNoticeIds(eventMessageFromChromeURL) {
+  for (const matchingContext of matchingContexts) {
+    const addWWWForBuildingURL = `www.${eventMessageFromChromeURL}`;
```

Comment:
Without www the regex does not seem to recognize the url. Should it still be recognized without www ?

Reply:
Actually, it should … And even, if we want to match on something.com but not on www.something.com I’m afraid you’re changing the condition … unless the URL you’re reading when visiting `www.example.com` in Chrome is `example.com` (because Chrome hides `www` ?)

Reply:
Yes chrome hides www this is why i'm adding it. I had some regex not recognizing example.com this is why i'm adding www

Reply:
The issue we may have is when we’re visiting `somesite.org` (no `www`) and we have matching context for `somesite.org` … We’re going to try the regex on `www.somesite.org` which will not match, right ? what do you think ?